### ISSUE TITLE
Allow anchorIdentifier under "tables"

### ISSUE THREAD

**Issue by ujovlado** [keboola/gooddata-writer-v3#47]
```
[2019-06-26 06:32:46] php-component.ERROR: Unrecognized option "anchorIdentifier" under "root.parameters.tables.out.c-xxx.yyy". Available options are "columns", "disabled", "grain", "identifier", "title". [] []
```

**Comment by kacurez** [comment]
podla kodu a dokumentacie by mal byt `anchorIdentifier` podporovany akurat neni v configDefinition
https://github.com/keboola/gooddata-writer-v3/blob/master/src/ConfigDefinition.php

btw v starom ani v novom ui gd writer sa anchorIdentifier nenastavuje resp nepracuje sa stym vobec
https://github.com/keboola/kbc-ui/search?q=anchorIdentifier&unscoped_q=anchorIdentifier

tak neviem ako sa tam dostal potom? :)

**Comment by ujovlado** [comment]
No bola to migracia. Takze asi migraciou zo stareho, kde to bolo zalozene.

**Comment by kacurez** [comment]
v migracii sa to explicitne nenastavuje, tam sa zoberie co je nastavene v starom konfigu a zkopiruje sa
https://github.com/keboola/config-migration-tool/search?q=anchorIdentifier&unscoped_q=anchorIdentifier
takze sa to nejak muselo dostat do stareho konfigu

**Comment by ujovlado** [comment]
Fixed in #48 :) Dik
